src/Components/Assets/AssetImportModal.tsx

```diff
@@ -31,24 +32,28 @@ const AssetImportModal = ({ open, onClose, facility }: Props) => {
     useState<(AssetData & { notes?: string; last_serviced_on?: string })[]>();
   const [location, setLocation] = useState("");
   const [locations, setLocations] = useState<any>([]);
-  const dispatchAction: any = useDispatch();
   const { sample_format_asset_import } = useConfig();
 
   const closeModal = () => {
     setPreview(undefined);
     setSelectedFile(undefined);
     onClose && onClose();
   };
+  const {
+    data: facilityAssetLocations,
+    loading,
+    refetch,
+  } = useQuery(routes.listFacilityAssetLocation, {
+    pathParams: { facility_external_id: `${facility.id}` },
```

Comment:
why string interpolation?

Reply:
pathparams need string value and facility.id is number | undefined